### DISCUSSION TITLE
Crash UTM with PowerPC

### DISCUSSION THREAD

**Discussion by Yoshi40a** [utmapp/UTM#6229] [UTM Help (iOS)]
**Power PC,512MB,20GB**
* UTM SE 4.4.1
* IPadOS 17.3
* IPad 9
* No
* IPA File(Sideloadly)

**Debug log**

qemu-system-ppc -L "/private/var/containers/Bundle/Application/AAA0BDA7-9AC5-4781-ADE6-89AEB6F0E9F4/UTM SE.app/qemu" -S -spice unix=on,addr=7C260C93-8230-414F-9BF3-36153342E37B.spice,disable-ticketing=on,image-compression=off,playback-compression=off,streaming-video=off,gl=off -chardev spiceport,id=org.qemu.monitor.qmp,name=org.qemu.monitor.qmp.0 -mon chardev=org.qemu.monitor.qmp,mode=control -nodefaults -vga none -device sungem,mac=C6:F8:CC:03:A4:E2,netdev=net0 -netdev user,id=net0 -device VGA -smp cpus=1,sockets=1,cores=1,threads=1 -machine mac99,via=pmu -accel tcg,thread=multi,tb-size=128 -m 512 -usb -device usb-tablet,bus=usb-bus.0 -device usb-mouse,bus=usb-bus.0 -device usb-kbd,bus=usb-bus.0 -device ide-hd,bus=ide.0,drive=drive1,bootindex=0 -drive "if=none,media=disk,id=drive1,file=/private/var/mobile/Containers/Data/Application/844F4D21-B6A2-434C-8B28-ABB116E58BD8/Documents/Mac OS 9.2.1.utm/Data/disk-0.qcow2,discard=unmap,detect-zeroes=unmap" -device ide-cd,bus=ide.1,drive=drive0,bootindex=1 -drive if=none,media=cdrom,id=drive0,readonly=on -device virtio-serial -device virtserialport,chardev=org.qemu.guest_agent,name=org.qemu.guest_agent.0 -chardev spiceport,id=org.qemu.guest_agent,name=org.qemu.guest_agent.0 -name "Mac OS 921" -uuid 7C260C93-8230-414F-9BF3-36153342E37B -rtc base=localtime

**Upload VM**
UTM Gallery (Mac OS 9.2.1)

**Comment by IMTHECOOLEST** [comment]
this is because your ios version utm wont work with 17.2+ for me, just use dosbox for rhapsody x86 ver its called iDos

**Comment by IMTHECOOLEST** [comment]
just DONT update your iphone, apple is getting smarter....

**Comment by IMTHECOOLEST** [comment]
its just useless now

**Comment by IMTHECOOLEST** [comment]
just uuse a pc for qemu
